tests/test_discoverer.py

```diff
@@ -73,3 +74,31 @@ def test_discover_removed_file():
     discoverer_test = discoverer_run_once()
     discoverer_test.start()
     assert int(max_db_id()) == 1
+
+
+# Trigger manual rescan
+def test_manual_rescan():
+    # Setup
+    shutil.copy2(file_path + origin_file, file_path + destination_file)
+    process_handle = subprocess.Popen(
+        "python -m flumes.discoverer -d " + file_path + " -i " + db_path, shell=True
+    )
+    time.sleep(6)
+    os.rename(file_path + destination_file, file_path + "sample-file2.mp4")
+    # Test
+    subprocess.run("kill -USR1 " + str(process_handle.pid), shell=True)
+    time.sleep(6)
+    db_select_result = subprocess.run(
+        ["sqlite3", db_name, "select MAX(id) from files;"],
+        capture_output=True,
+        text=True,
+        cwd="tests",
+    )
```

Comment:
Instead of checking the max id in the db it's safer to get the name of the entry with id 2 and compare it against the expected filename in that position